esinstall/src/rollup-plugins/rollup-plugin-wrap-install-targets.ts

```diff
@@ -37,7 +37,7 @@ export function rollupPluginWrapInstallTargets(
    */
   function cjsAutoDetectExportsTrusted(normalizedFileLoc: string): string[] | undefined {
     try {
-      const mod = require(normalizedFileLoc);
+      const mod = eval('require')(normalizedFileLoc);
```

Comment:
Can you leave a comment explaining why this is needed? This scares me lol

Reply:
tbh same, I have using eval. Webpack offers a fallback (_non_webpack_require_ or something) but Rollup doesn't, so as far as I can tell this is the official way to get runtime `require` calls in Rolly.

fwiw I saw @developit doing this as well so if we're committing JS crime at least we're in good company. I'll add a comment regardless.

Reply:
> if we're committing JS crime at least we're in good company

😂 AKA “JavaScript Development“